src/error.rs

```diff
@@ -14,20 +14,21 @@
 extern crate serde_json;
 
 
+
+use ceph_version::CephVersion;
 use serde_json::error::Error as SerdeJsonError;
 use std::{fmt, str};
 use std::error::Error as StdError;
-use std::ffi::{IntoStringError, NulError};
+use std::ffi::{FromBytesWithNulError, IntoStringError, NulError};
 use std::io::Error;
 use std::num::ParseIntError;
 use std::string::FromUtf8Error;
 use uuid::ParseError;
 
-use ceph_version::CephVersion;
-
 /// Custom error handling for the library
 #[derive(Debug)]
 pub enum RadosError {
+    FromBytesWithNulError(FromBytesWithNulError),
```

Comment:
Where is this being used?

Reply:
Nowhere.  I forgot to delete this after some experiments.